test/integration/test_w_valgrind.py

```diff
@@ -0,0 +1,45 @@
+# coding=UTF-8
+
+"""
+Tests the representation of asbackup/asrestore output runned with valgrind
```

Comment:
Suggestion: maybe change the comment to something like "Tests basic asbackup/asrestore commands with valgrind for memory leaks/errors."

Reply:
done.